crossref.py

```diff
@@ -18,23 +19,33 @@ def handleData(response):
         file = csv.DictWriter(csvfile, fieldnames = fieldnames)
         file.writeheader()
 
-        # get all entries
-        for i in range(1, int(rows), 1):
-
-            # read the metadata of one event from response
-            metadata = response['message']['items'][i]['event']
-
-            # add the title and doi of the proceeding
-            # TODO: .replace('\n', '')
-            metadata['proceedings-title'] = ''.join(response['message']['items'][i]['title']).encode('utf-8')
-            metadata['doi'] = response['message']['items'][i]['DOI']
-
-            # TODO: fix encoding
-            [''.join(x).encode('utf-8') for x in metadata ]
-            print (metadata)
-
-            # write metadata to file
-            file.writerow(metadata)
+        print('handleData', response['message'].keys())
```

Comment:
Should this be indented inside the for loop? @andrecastro0o

Reply:
no. It is correct like that. is so we see the keys of response['message'], which the loop will loop over